applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -173,6 +248,54 @@ static uint32_t get_and_incr_seq_num(const struct bt_audio_stream *stream)
 	return 0;
 }
 
+static int update_sink_stream_qos(uint32_t pres_delay_us)
+{
+	int ret;
+
+	lc3_preset_sink.qos.pd = pres_delay_us + 10;
+
+	for (int i = 0; i < ARRAY_SIZE(headsets); i++) {
+		LOG_INF("Headset %d, connection", i);
+
+		if (headsets[i].headset_conn != NULL && headsets[i].sink_stream.ep != NULL) {
+			headsets[i].sink_stream.qos->pd = pres_delay_us + 20;
+			headsets[i].sink_stream.ep->qos.pd = pres_delay_us + 30;
```

Comment:
Seems that the qos can be updated whilst a stream is running?

Reply:
No, these do not cause qos update only puts the new value in place ready for the stop and reconfigure.